drivers/adc/ad717x/ad717x.c

```diff
@@ -52,6 +52,333 @@
 #define COMM_ERR    -2 /* Communication error on receive */
 #define TIMEOUT     -3 /* A timeout has occured */
 
+/***************************************************************************//**
+ * @brief Set channel status - Enable/Disable
+ * @param device - AD717x Device descriptor.
+ * @param channel_id - Channel ID (number) of the channel whose status is to be set.
+ * @param channel_status - Required status of the channel-True in case of Enable
+ *			    	and False in case of Disable
+ * @return Returns 0 for success or negative error code in case of failure.
+*******************************************************************************/
+int ad717x_set_channel_status(ad717x_dev *device, uint8_t channel_id,
+			      bool channel_status)
+{
+	ad717x_st_reg *chn_register;   // Pointer to channel register
+	int ret;	    // Variable to return the status
```

Comment:
No need for these comments. The lines are quite self-explanatory.

Reply:
Removed comments related to variables